test/fixtures/responses/get-transactions-one.json

```diff
@@ -5,7 +5,7 @@
     "sequence": 491,
     "id": "D868CFF0DF8C8AAF205404460EA764ACB3B8862527FA414BC8C1CA9A45B1F276",
     "specification": {
-      "domain": "ripple.com"
+      "domain": "example.com"
```

Comment:
I'm not sure how this fixture is used; this line may need to change back.

Reply:
`ripple.com` in hex is `726970706C652E636F6D` and `example.com` is `6578616D706C652E636F6D`. I will add a commit that replaces all of the former with the latter

Reply:
Looks like tests are failing because of this change, so I think it'll have to be added to this PR